doc/Manifest.toml

```diff
@@ -26,7 +26,7 @@ version = "0.8.6"
 deps = ["ANSIColoredPrinters", "Base64", "Dates", "DocStringExtensions", "IOCapture", "InteractiveUtils", "JSON", "LibGit2", "Logging", "Markdown", "REPL", "Test", "Unicode"]
 git-tree-sha1 = "2c023382ab49c40475fcf59b90ba1c8edd9ff45e"
```

Comment:
Don't you need to change the tree hash as well?

Reply:
See the top of this file:

> # This file is machine-generated - editing it directly is not advised

:)

Reply:
Thanks